### PR TITLE
[INLONG-7092][Dashboard] Source and Sink as sub-concepts of Stream

### LINKED ISSUE #7092: [Feature][Dashboard] Source and Sink as sub-concepts of Stream
### Description

_No response_

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by leezng**
- Fixes #7092
image

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

**Review by bluewang — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -489,8 +489,8 @@
   "pages.GroupDetail.Sources.status.BeenRetry": "已下发重试",
   "pages.GroupDetail.Sources.status.BeenFrozen": "已下发停止",
   "pages.GroupDetail.Sources.status.BeenActive": "已下发重启",
-  "pages.GroupDetail.Sink.New": "新建流向配置",
-  "pages.GroupDetail.Sink.Edit": "编辑流向配置",
+  "pages.GroupDetail.Sink.New": "新建目标",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:492]: 目标
->
数据目标

File: inlong-dashboard/src/locales/cn.json [hunk 2]
```diff
@@ -489,8 +489,8 @@
...
+  "pages.GroupDetail.Sink.Edit": "编辑目标",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:493]: 目标
->
数据目标

**Review by healchow — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json (hunk 1 above)
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:492]:
```suggestion
  "pages.GroupDetail.Sink.New": "新建数据目标",
```

File: inlong-dashboard/src/locales/cn.json (hunk 2 above)
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:493]:
```suggestion
  "pages.GroupDetail.Sink.Edit": "编辑数据目标",
```

**Pull request merged**